src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -214,7 +214,13 @@ def get_tags(
     ):
         default_tags = set()
 
-    tags = tags | default_tags
+    use_improved_block_kit = features.has(
+        "organizations:slack-block-kit-improvements", group.project.organization
+    )
+    # improved block kit only uses alert rule tags
+    if not use_improved_block_kit:
+        tags = tags | default_tags
```

Comment:
tags are passed in from the alert rule. if we're under the EA'd feature flag, we still use default tags + alert rule tags. if we're under the LA feature flag (sentry only ATM) then we only use the alert rule tags